float32 x_acceleration
float32 y_acceleration
float32 z_acceleration
